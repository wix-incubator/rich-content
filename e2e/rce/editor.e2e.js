describe('editor', () => {
  it('should support lists', () => {
    cy.editorDesktop();

    cy.matchContentSnapshot('before');

    cy.enterText('hello{selectall}');
    cy.get('[data-hook=textInlineStyleButton_Bold]').click();

    cy.matchContentSnapshot('after');
    cy.matchImageSnapshot();
  });
});
