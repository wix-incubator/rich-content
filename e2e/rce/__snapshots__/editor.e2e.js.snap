exports[`editor > should allow to enter text #0`] =
{
  "blocks": [
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 0,
      "text": "Leverage agile frameworks",
      "type": "unstyled"
    },
    {
      "data": {},
      "depth": 0,
      "entityRanges": [],
      "inlineStyleRanges": [],
      "key": 1,
      "text": "to provide a robust synopsis for high level overviews.",
      "type": "unstyled"
    }
  ],
  "entityMap": {}
};
